icefoganalytics/travel-authorization
# Refactor: Rename `web/apis` Folder to `web/api`

# Context

Make space for future apis, and hope to avoid confusion.

Why? Because this folder represents _the_ api mapping to the back-end, rather than a collection of apis to various locations.

If there was an api that corresponded to something else, such as auth0, then it would go in a folder named `web/auth0-api/`

# Testing Instructions

1. Boot the app via `dev up`.
2. Log in at http://localhost:8080.
3. Check that you can view/create/update travel authorizations under the "My Travel Requests" top nav menu item.
4. Check that you can create/read/update/delete estimates on a "draft" travel authorization.
